raiden/ui/cli.py

```diff
@@ -978,10 +984,10 @@ def _run_app(self):
             try:
                 api_server.start(api_host, api_port)
             except APIServerPortInUseError:
-                print(
-                    'ERROR: API Address %s:%s is in use. '
-                    'Use --api-address <host:port> to specify port to listen on.' %
-                    (api_host, api_port),
+                click.secho(
+                    f'ERROR: API Address {api_host}:{api_port} is in use. '
+                    f'Use --api-address <host:port> to specify a different port.',
+                    fg='red',
```

Comment:
At this point, the `api_server.start` exception is being handled and `api_server` was not added yet to the `tasks` list. In the line below, you should clean the already started stasks (as done by that `stop_task`+`joinall` in the end of this function), and **then** exit.